### ISSUE TITLE
spi bus help

### ISSUE THREAD

**Issue by natrow** [ryankurte/embedded-hal-compat#18]
I'm trying to use this crate to work with the SPI implementation from the stm32l4xx_hal crate (uses embedded_hal 0.2.7) and the SpiBus trait from embedded_hal-1.0.0-alpha.10 but when I use .forward() I get the following error:

```
error[E0277]: the trait bound `embedded_hal_compat::forward::ForwardError<stm32l4xx_hal::spi::Error>: From<embedded_hal_compat::forward::ForwardError<nb::Error<stm32l4xx_hal::spi::Error>>>` is not satisfied
  --> src/lib.rs:80:17
   |
80 |             foo(spi_bus);
   |             --- ^^^^^^^ the trait `From<embedded_hal_compat::forward::ForwardError<nb::Error<stm32l4xx_hal::spi::Error>>>` is not implemented for `embedded_hal_compat::forward::ForwardError<stm32l4xx_hal::spi::Error>`
   |             |
   |             required by a bound introduced by this call
   |
   = help: the trait `SpiBus` is implemented for `Forward<T>`
   = note: required for `Forward<Spi<SPI1, (Pin<Alternate<PushPull, 5>, L8, 'A', 5>, Pin<Alternate<PushPull, 5>, L8, 'A', 6>, ...)>>` to implement `SpiBus`
note: required by a bound in `foo`
  --> src/lib.rs:27:14
   |
25 |     fn foo<BUS>(_: BUS)
   |        --- required by a bound in this function
26 |     where
27 |         BUS: embedded_hal_compat::eh1_0::spi::SpiBus,
   |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `foo`

For more information about this error, try `rustc --explain E0277`.
```

The bus does implement the FullDuplex<u8> trait from embedded_hal-0.2.7 so I don't understand why calling `.forward()` doesn't work...

**Comment by eldruin** [comment]
Thanks for reporting! I have fixed this in #19

**Comment by eldruin** [comment]
The fix for this has been released in version 0.10.1.
